src/server/dfly_main.cc

```diff
@@ -299,6 +300,21 @@ string NormalizePaths(std::string_view path) {
   return string(path);
 }
 
+bool StoiOctalBase(string str, int* out) {
```

Comment:
Please use `safe_strtou32_base` from absl. (yeah, it's internal, but lets still use it).